src/lib/components/Ideogram.react.js

```diff
@@ -450,10 +445,10 @@ Ideogram.propTypes = {
     chrWidth: PropTypes.number,
 
     /**
-     * A list of the names of chromosomes to 
-     * display. Useful for depicting a subset of the chromosomes in the genome, 
+     * A list of the names of chromosomes to
+     * display. Useful for depicting a subset of the chromosomes in the genome,
      * e.g. a single chromosome.
```

Comment:
```suggestion
     * e.g., a single chromosome.
```
And it looks like the above three lines should fit over two lines.